api/Controllers/LoginController.cs

```diff
@@ -8,31 +8,33 @@
 
 namespace api.Controllers
 {
-    [Route("api/[controller]")]
-    [ApiController]
-    public class LoginController: ControllerBase
+  [Route("api/[controller]")]
+  [ApiController]
+  public class LoginController : ControllerBase
+  {
+    private readonly ILoginService _loginService;
+    private readonly IMapper _mapper;
+    private readonly IConfiguration _configuration;
+
+    public LoginController(ILoginService loginService, IMapper mapper, IConfiguration configuration)
+    {
+      _loginService = loginService;
+      _mapper = mapper;
+      _configuration = configuration;
+    }
+
+    [HttpPost]
+    [AllowAnonymous]
+    public async Task<IActionResult> Post([FromBody] UserLoginDto loginUser)
```

Comment:
This method has been renamed to Post and should be renamed back since Login is more appropriate - it describes the action. Also, the document needs to be formatted correctly - it has 2 space tabs instead of 4.

Reply:
Auto formatting does not work whatsoever, but I renamed the method back